kumascript/macros/ServiceWorkerSidebar.ejs

```diff
@@ -26,6 +26,22 @@ var text = mdn.localStringMap({
     'Channel_Messaging_API': 'Channel Messaging API',
     'Web_Workers_API': 'Web Workers API'
   },
+  'fr': {
+    'Service_Worker_API': 'API Service Worker',
+    'Overview': 'Aperçu des interfaces Service Worker',
+    'Service_Worker_guides': 'Guides Service Worker',
+    'Interfaces': 'Interfaces',
+    'Documentation': 'Documentation&nbsp;:',
```

Comment:
I believe `nbsp;`s don't work here. `\xa0` (or `\u00a0`) seems to work on my local machine:

```suggestion
    'Documentation': 'Documentation\xa0:',
```